apps/state/config/config.exs

```diff
@@ -89,6 +89,9 @@ config :state, :shape,
     "858t0001" => -1,
     # Green-D (Newton Highlands)
     "858t0002" => -1,
+    # Green-E (shuttle bus)
+    "6020021" => -1,
+    "6020022" => -1,
```

Comment:
So if I'm understanding this correctly, `prefix_overrides` just overrides the priority of certain shapes, and this is making sure that the shuttle shapes are deprioritized relative to the normal ones?

Reply:
Right. https://github.com/mbta/api/blob/7ce0886a643bc09f79388779142c40349b730f28/apps/state/config/config.exs#L150-L152 has a general rule like this for shuttle shapes that we build ourselves, but this one comes out of HASTUS so we need to do a specific override.